### PR TITLE
#138 Plot predictors forecasts in predictor_robojudge.ipynb

### PR DESCRIPTION
**Pull request by ofrancon**
Updated `predictor_robjojudge` notebook to plot forecasts to make it easy to compare predictors against each other, as well as against ground truth.

### CODE REVIEW

File: predictor_robojudge.ipynb [hunk 1]
```diff
@@ -728,10 +801,12 @@
...
     "# Add 1 trace for the true number of cases\n",
     "for geoid_name in geoid_names:\n",
+    "    # Only plot data that is known, for ground truth\n",
+    "    last_known_date = actual_df[actual_df.ActualDailyNewCases7DMA.notnull()].Date.max()\n",
     "    geo_actual_df = actual_df[(actual_df.GeoID == geoid_name) &\n",
     "                                  (actual_df.Date >= start_date)]\n",
-    "    fig.add_trace(go.Scatter(x=geo_actual_df.Date,\n",
-    "                             y=geo_actual_df.ActualDailyNewCases7DMA,\n",
+    "    fig.add_trace(go.Scatter(x=geo_actual_df[actual_df.Date <= last_known_date].Date,\n",
+    "                             y=geo_actual_df[actual_df.Date <= last_known_date].ActualDailyNewCases7DMA,\n",
```
Comment (ofrancon) [inline predictor_robojudge.ipynb:809]: Plot ground truth for each country only until last know date

File: predictor_robojudge.ipynb [hunk 2]
```diff
@@ -741,8 +816,9 @@
     "# Add 1 trace for the overall ground truth\n",
     "overall_actual_df = actual_df[actual_df.Date >= start_date].groupby([\"Date\"])[[\"GeoID\", \"ActualDailyNewCases7DMA\"]].sum(). \\\n",
     "    sort_values(by=[\"Date\"]).reset_index()\n",
-    "fig.add_trace(go.Scatter(x=overall_actual_df.Date,\n",
-    "                         y=overall_actual_df.ActualDailyNewCases7DMA,\n",
+    "overall_last_known_date = overall_actual_df[overall_actual_df.ActualDailyNewCases7DMA > 0].Date.max()\n",
+    "fig.add_trace(go.Scatter(x=overall_actual_df[overall_actual_df.Date <= overall_last_known_date].Date,\n",
```
Comment (ofrancon) [inline predictor_robojudge.ipynb:820]: Plot the overall ground truth only until last know date

File: covid_xprize/scoring/predictor_scoring.py [hunk 3]
```diff
@@ -66,5 +66,5 @@ def add_population_column(df):
     """
 
     pop_df = XPrizePredictor._load_additional_context_df()
-    return df.merge(pop_df, on=['GeoID'], how='left', suffixes=('', '_y'))
+    return df.merge(pop_df[['GeoID', 'Population']], on=['GeoID'], how='left', suffixes=('', '_y'))
```
[the remark this answers is not in the document]
Comment (EKMeyerson) [reply covid_xprize/scoring/predictor_scoring.py:69]: love this

**Review by EKMeyerson — APPROVED** [review]
lgtm

**Pull request merged**